ci-automation/ci_automation_common.sh

```diff
@@ -299,3 +300,36 @@ function secret_to_file() {
     config_ref="/proc/${$}/fd/${fd}"
 }
 # --
+
+# :
+function sign_artifacts() {
+    local signer="${1}"; shift
+    # rest of the parameters are directories/files to sign
+    local to_sign=()
+    local file
+    local files
+
+    if [[ -z "${signer}" ]]; then
+        return
+    fi
```

Comment:
How about handling `[ "$*" = "*" ]` when there were no files and the glob didn't expand?

Reply:
Maybe also for the case where `*` is the last character of the string like `sometarball*` when the caller uses `"${dest_tarball}"*`